.swiftlint.yml

```diff
@@ -5,14 +5,45 @@ excluded:
   - DerivedData
   - Pods
 
+analyzer_rules:
+  - unused_import
+
 disabled_rules:
+  - block_based_kvo
+  - closure_body_length
+  - computed_accessors_order
+  - cyclomatic_complexity
+  - duplicate_imports
+  - empty_enum_arguments
+  - empty_string
+  - file_length
+  - for_where
+  - force_cast
+  - force_try
+  - force_unwrapping
+  - function_body_length
   - function_parameter_count
   - identifier_name
+  - implicit_getter
+  - implicitly_unwrapped_optional
+  - indentation_width
+  - large_tuple
+  - legacy_objc_type
   - line_length
   - multiple_closures_with_trailing_closure
   - nesting
+  - orphaned_doc_comment
+  - operator_whitespace
+  - return_arrow_whitespace
+  - shorthand_operator
   - todo
+  - trailing_closure
+  - type_body_length
   - type_name
+  - unneeded_break_in_switch
+  - unused_optional_binding
+  - vertical_parameter_alignment
+  - xctfail_message
```

Comment:
Thanks for the effort!
I think it will be easier to fix issues one by one.

Reply:
I'll resume fixes as soon as this PR is merged but for now, there are a few questions.